src/fast_check/transform.rs

```diff
@@ -1821,6 +2368,15 @@ fn is_expr_ident_or_member_idents(expr: &Expr) -> bool {
   }
 }
 
+fn contains_return_stmt(stmts: &Vec<Stmt>) -> bool {
+  for stmt in stmts {
+    if let Stmt::Return(_) = stmt {
+      return true;
+    }
+  }
+  false
+}
```

Comment:
This will return false for:

```ts
if (true) {
  return true;
} else {
  return false;
}
```

We can use `get_return_stmts_with_arg_from_stmts`, which already exists to figure this out.